srcCpp/qos_string.h

```diff
@@ -1,53 +1,53 @@
 #include <string.h>
 #define PERFTEST_QOS_STRING_SIZE 46
-#define PERFTEST_QOS_STRING_TOTAL_SIZE 46339
+#define PERFTEST_QOS_STRING_TOTAL_SIZE 46386
```

Comment:
This value could be calculated instead of changing this manually as:
```
const unsigned int PERFTEST_QOS_STRING_TOTAL_SIZE = sizeof(PERFTEST_QOS_STRING) / sizeof (char *);
```